api/dedibox/v1/dedibox_utils.go

```diff
@@ -0,0 +1,99 @@
+package dedibox
+
+import (
+	"github.com/scaleway/scaleway-sdk-go/internal/async"
+	"github.com/scaleway/scaleway-sdk-go/internal/errors"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"time"
+)
+
+const (
+	defaultRetryInterval = time.Second * 15
+	defaultTimeout       = time.Second * 30
```

Comment:
30s isnt a bit too low for a Dedibox server creation ?

Reply:
Oups went too fast means minutes